crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -3481,7 +3481,7 @@ fn get_quoted_utf16(v: &str, ascii_only: bool, target: EsVersion) -> String {
 
                 match next {
                     Some('1'..='9') => buf.push_str("\\x00"),
-                    _ => buf.push_str("\\0"),
+                    _ => buf.push_str("\\x00"),
```

Comment:
I think the only problem was the range here:
```suggestion
                    Some('0'..='7') => buf.push_str("\\x00"),
                    _ => buf.push_str("\\0"),
```

Reply:
Thank you!